temporalio/worker/_workflow.py

```diff
@@ -552,6 +552,10 @@ def _create_workflow_instance(
             priority=temporalio.common.Priority._from_proto(init.priority),
         )
 
+        previous_run_failure = None
+        if init.HasField("continued_failure"):
+            previous_run_failure = init.continued_failure
```

Comment:
nit: up to you but personally I'd write

```python
previous_run_failure = init.continued_failure if if init.HasField("continued_failure") else None
```

I think the slightly-more functional style is well suited to Python, or IOW I dislike Go's / C's procedural style of "create some transient inconsistent state then fix it"